doc/source/testing.rst

```diff
@@ -23,7 +23,7 @@ This is a set of environment variables used by the testing infrastructure.
   it acts the same as setting ``XDG_CACHE_HOME``. This overrides the default
   from :func:`papis.utils.get_cache_home` that uses ``platformdirs``.
 * ``PAPIS_DATABASE_BACKEND``: set to one of the supported database backends. Most
-  existing tests do no parametrize over the database backend, so this value lets
+  existing tests do not parametrize over the database backend, so this value lets
```

Comment:
Attention here: I notice that this has a typo, but I didn't really understand the meaning of the sentence. Is this good like this?

Reply:
Yeah, that's right!